ruby/vampires.rb

```diff
@@ -0,0 +1,91 @@
+#name
+# age / where were you born
+
+# would you like some garlic bread'
+
+# would you like to enroll in the company health plan
+
+puts "How many employees will be processed?"
+number_of_employees = gets.to_i
+
+until number_of_employees == 0 
+
+  puts "Please enter your name (First Last)."
+  employee_name = gets.chomp
+
+
+  puts "Please enter your age."
+  age = gets.to_i
+
+  puts  "What year were you born (xxxx)?"
+  birth_year =  gets.to_i
+
+  if age == 2017 - birth_year 
+    age = true 
+  else 
+    age = false
+  end
+
+  puts  "Would you like some garlic bread (yes,no)?"
+  garlic_bread = gets.chomp
+
+  if garlic_bread == "yes"
+    garlic_bread = true
+
+  else 
+    garlic_bread = false
+
+  end
+
+  puts  "Would you be interested in enrolling in the company health plan(yes,no)?"
+
+  health_plan = gets.chomp
+
+  if health_plan == "yes"
+    health_plan = true
+  else
+    health_plan = false
+  end
+
+  if age && (garlic_bread|| health_plan)
+    puts "Probably not a vampire"
+
+  elsif !age && (!garlic_bread || !health_plan)
+    puts "Probably a vampire"
+
+  elsif
+    !age && !garlic_bread && !health_plan
+    puts "Probably a vampire"
+
+  elsif
```

Comment:
Consider the optimum condition when running your code.  What can we do to run only necessary processes?  We know that if the employee has Drac or Fang in their name then they are definitely a vampire.  If that is the case, do we need to run any of the other checks?

Consider:

```
if name_matches_vampire_params
  "Definitely a vampire"
else
  if #other checks
    "Something"
  elsif #more conditions...

  else
  end
end
```